### ISSUE TITLE
[BUG]Sub-agents incorrectly displayed at the same hierarchy level as main agents, when user imports Agents

### ISSUE THREAD

**Issue by liana-gif** [ProjectAlita/projectalita.github.io#521]
**Description:**
In the "Import parameters" screen, sub-agents (ADO Board and SQL Task) are displayed at the same hierarchy level as main agents (Recheck multi). According to the design, sub-agents should be nested under the main hierarchy to maintain a clear organizational structure.

**Steps to Reproduce:**

Open the "Import parameters" screen.
Expand the main agent hierarchy, such as Recheck multi.
Observe the hierarchy structure of sub-agents like ADO Board and SQL Task.
Expected Result:
Sub-agents should be displayed as nested elements under their respective main agent hierarchy.

**Actual Result:**
Sub-agents (ADO Board and SQL Task) appear at the same level as main agents instead of being nested within the main agent hierarchy.

**Screenshot:**
Refer to the attached screenshot for visual reference.![Image](https://github.com/user-attachments/assets/ac26c055-ab8c-4962-8b6c-d73966dbb64e)

**Comment by HawkQing1** [comment]
@liana-gif, no, they are not sub agents, they are just referred by tools of other agents and they will be imported if the user keeps the tools not changing. I don't think this is a bug.

**Comment by HawkQing1** [comment]
It will not useful if we have too many entities tools
